Darcmarc78/steam-reaction
# Add "jest" to project

# Overview
Need to add a testing framework

# Solution
Added `jest` to project. Wrote and tested a sample test

# Test Plan
1. Pull branch to local environment
2. Open a terminal in the local repository, enter `npm i` or `npm install`
3. Run `npm test` to see `jest` 
6. Open `localhost:8000/` in a browser.
